app/packages/looker/src/state.ts

```diff
@@ -132,6 +132,7 @@ interface BaseOptions {
   coloring: Coloring;
   customizeColorSetting: CustomizeColor[];
   selectedLabels: string[];
+  attributeVisibility: object;
```

Comment:
This should have a type. Can it be added, or a todo comment?

Reply:
Conceptually, visibility could be built into the `filter` function and this option wouldn't be necessary. This seems like it could be decoupled from `Looker`

Reply:
@benjaminpkane  We are actually using the `filter` function to determine which tag/overlay to render. I just need a flag to determine whether to reload the overlays for visibility changes because filters always trigger grids reload. That's why I added `attributeVisibility` separately.

Reply:
Ok I read the code in `tags.ts`. I see why it's used. At some point we will need to move `tags.ts` to canvas to avoid equality checks and formalize rendering as an overlay only feature.